src/plugin/luapi_application.h

```diff
@@ -867,6 +876,132 @@ static int applib_getDisplayDpi(lua_State* L) {
 }
 
 
+/**
+ * Exports the current document as a pdf or as a svg or png image
+
+ * Example 1:
+ * app.export({["outputFile"] = "Test.pdf", ["range"] = "2-5; 7", ["background"] = "none", ["progressiveMode"] = true})
```

Comment:
A quick documentation comment that describes what `progressiveMode` is could be helpful here! (Alternatively, this could be documented on the website/in an example plugin).

Reply:
Done (in Example 1)